dockerfiles/fedora

```diff
@@ -0,0 +1,14 @@
+from fedora:latest
+
+RUN dnf install -y -q make cmake git gcc-c++ gcc-gfortran flex patch
+
+RUN groupadd -r flecsi && useradd -r -m -g flecsi flecsi
+USER flecsi
+
+WORKDIR /tmp
+RUN git clone --recursive --depth=1 https://github.com/losalamos/flecsi-third-party tpl; \
+    mkdir -p tpl/build && cd tpl/build; \
+    cmake -DENABLE_EXODUS=ON -DENABLE_LAPACK=ON \
+          -DENABLE_METIS=ON  -DENABLE_SCOTCH=ON \
+          -DENABLE_LEGION=ON -DCMAKE_INSTALL_PREFIX=$HOME/tpl .. && make -j2; \
+    cd -;
```

Comment:
Do we really want that kind of build process? I assume that we will push to the Flecsi repo frequently enough that will trigger the "build Flecsi after new TPL" without doing it here.